test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -436,11 +436,13 @@ describe('CSOT spec prose tests', function () {
        *    - Expect this to fail with a timeout error.
        * 1. Verify that an `aggregate` command and two `getMore` commands were executed against the `db.coll` collection during the test.
        */
-      it.skip('sends correct number of aggregate and getMores', metadata, async function () {
+      it('sends correct number of aggregate and getMores', metadata, async function () {
+        // NOTE: we don't check for a non-zero ID since we lazily send the initial aggregate to the
```

Comment:
Our version of this would be to use `await cursorInit(changeStream)` (or `csInit`, can't remember the exact name) to force the initialization of the change stream without iterating it.  Then we can check its id.  Is there a reason we don't do that here?

Reply:
No reason in particular, I can do that here.